tests/jerry/es2015/object-prototype-proto.js

```diff
@@ -54,5 +54,33 @@ try {
   assert(e instanceof TypeError);
 }
 
-var o2 =  { ["__proto__"] : null };
+var o2 = { ["__proto__"] : null };
 assert(o2.__proto__ === null);
+assert(Object.getPrototypeOf(o2) === Object.prototype);
+
+var o3 = { __proto__ : null };
+assert(o3.__proto__ === undefined);
+assert(Object.getPrototypeOf(o3) === null);
+
+var o4 = { "__proto__" : null };
```

Comment:
There should be a test when the value is set to an existing object. Just create a `var o = {}`